src/system_descriptor.jl

```diff
@@ -87,7 +85,7 @@ function get_supported_args_permutations(v::SystemDescriptor)
             comprehensive_set = union(comprehensive_set, set)
         end
 
-        for values in product(Iterators.repeated(comprehensive_set, length(keys_arr))...)
+        for values in product(repeated(comprehensive_set, length(keys_arr))...)
```

Comment:
We typically write `Iterators.product` and `Iterators.repeated`. There are likely some exceptions for certain types.